app-web/caddy/autobuild/postinst

```diff
@@ -0,0 +1,4 @@
+systemd-sysusers caddy.conf
```

Comment:
Add some `echo` commands to tell the user what is happening

Reply:
systemd-sysusers already have some output
```
Unpacking caddy (2.7.6) ...
Setting up caddy (2.7.6) ...
Creating group 'caddy' with GID 984.
Creating user 'caddy' (Caddy web server) with UID 984 and GID 984.
  SUCCESS oma has successfully applied changes to your system.
     INFO If you would like to undo these changes, please use the `oma undo' 
          command.
```

Reply:
That's different: the user can easily identify which output belongs to which command.